charts/operator-wandb/values.yaml

```
@@ -862,6 +870,17 @@
         optional: true
+    - name: mysql-certs
+      secret:
+        secretName: "{{ .Values.global.mysql.tlsSecret.name }}"
+        optional: true
+        items:
+          - key: "{{ .Values.global.mysql.tlsSecret.keyKeyName }}"
+            path: tls.key
+          - key: "{{ .Values.global.mysql.tlsSecret.crtKeyName }}"
+            path: tls.crt
+          - key: "{{ .Values.global.mysql.tlsSecret.caKeyName }}"
+            path: ca.crt 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)